package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "typeorm",
   "private": true,
-  "version": "0.0.9",
+  "version": "0.1.0-alpha.1",
```

Comment:
I know that it bad for users but I hope they understand that its hard to maintain such a complex big library to a single person **effectively**. So for now strategy is to try to make breaking changes only between 0.x versions and all other non breaking changes in 0.0.x versions.